### DISCUSSION TITLE
No module named ''netbox-secretstore'

### DISCUSSION THREAD

**Discussion by Tayberi** [DanSheps/netbox-secretstore#14] [Q&A] [answered]
I can't install the plugin netbox-secretstore.

Environment
Python version: 3.7
NetBox version: 3.0.2

source /opt/netbox/venv/bin/activate
pip install netbox-secretstore

configuration.py:
PLUGINS = ['netbox-secretstore',]

(venv) $ cd /opt/netbox/netbox/ (venv) $ python3 manage.py migrate

**Comment by Tayberi** [comment]
![photo_2021-09-13_17-35-44](https://user-images.githubusercontent.com/3882883/133192756-f5ff57fb-15c0-47ba-9c00-2b42d8fc01ec.jpg)

**Comment by DanSheps** [comment, marked as answer]
The plugin once installed netbox_secretstore.

This is similar to most django plugins out there that have a dash.

**Reply by Andiamorim** [reply]
Thanks, this helped me with another plugin as well.
Adjusting the "configuration.py" file, changing the "plugin-name" to "plugin_name".
